.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 .netlify
 .idea
+package-lock.json
```

Comment:
Somehow `package-lock.json` still in commit, could you delete it additionally?

Reply:
If I delete, then won't it delete it from the main branch when it is merged?

Reply:
I mean not this line. This line is good. I mean could you delete the actual file? Cause file still in changes, but it is unnecessary because we use `yarn.lock`  file instead

Reply:
Got it.